kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/config/TopicNameBasedKekSelector.java

```diff
@@ -19,10 +18,10 @@
 public abstract class TopicNameBasedKekSelector<K> {
 
     /**
-     * Returns a completion stage whose value, on successful completion, is a map from each of the given topic
-     * names to the KEK id to use for encrypting records in that topic.
+     * Returns a completion stage whose value, on successful completion, is a topic name selection containing the
+     * resolved key ids for topics which could be resolved, and a set of unresolved topic names.
      * @param topicNames A set of topic names
-     * @return A completion stage for the map form topic name to KEK id.
+     * @return A completion stage containing a topic name selection
```

Comment:
gah I should have kept the `for the` language, containing isn't right for a CompletionStage